src/NHibernate/Mapping/ByCode/SimpleModelInspector.cs

```diff
@@ -73,6 +72,11 @@ public bool IsManyToMany(MemberInfo member)
 				return ManyToManyRelations.Contains(member);
 			}
 
+			public bool IsManyToManyKey(MemberInfo member)
+			{
+				return ItemManyToManyRelations.Contains(member);
```

Comment:
There is a mistake. Should be KeyManyToManyRelations (which had been gone. See comment above.)